Sources/SentryCrash/Recording/Tools/SentryCrashJSONCodec.h

```diff
@@ -42,6 +42,8 @@ extern "C" {
  */
 #define SentryCrashJSON_SIZE_AUTOMATIC -1
 
+#define SentryCrashMAX_ReportSize 100000
```

Comment:
`m`: I think it's not the maximum size of the report, but the size of the buffer for decoding a JSON element.